packages/react/src/components/proxies.ts

```diff
@@ -72,6 +72,14 @@ import { defineCustomElement as defineIonToggle } from '@ionic/core/components/i
 import { defineCustomElement as defineIonToolbar } from '@ionic/core/components/ion-toolbar.js';
 import { defineCustomElement as defineIonVirtualScroll } from '@ionic/core/components/ion-virtual-scroll.js';
 
+const excludeValueProp = (_: any, props: any) => {
```

Comment:
This file is unfortunately autogenerated, so any changes here will be overwritten every time the `core` package is built.

Instead, we will need to exclude `ion-select` from being auto generated here: https://github.com/ionic-team/ionic-framework/blob/main/core/stencil.config.ts#L72

and then create a new file in `packages/react/src/components/` called `IonSelect.tsx` that is basically:

```tsx
import type { JSX } from '@ionic/core/components';
import { defineCustomElement  } from '@ionic/core/components/ion-select.js';

import { createReactComponent } from './react-component-lib';

const excludeValueProp = (_: any, props: any) => {
  const newProps = {
    ...props
  };
  delete newProps['value'];
  return newProps;
}

export const IonSelect = /*@__PURE__*/createReactComponent<JSX.IonSelect, HTMLIonSelectElement>('ion-select', undefined, excludeValueProp, defineCustomElement);
```

and finally export it from `packages/react/src/components/index.ts`.

Reply:
Since React reflects props to attributes does it make sense for the `render` function to filter out any prop with an object value to avoid this potential kind of issue elsewhere?